spec/support/tom_select_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module TomSelectHelper
```

Comment:
There are already some helper for tom select here :
https://github.com/openfoodfoundation/openfoodnetwork/blob/dd502f0883d8099edec197d3368eb5d9420669ea/spec/support/request/web_helper.rb#L93
But like the idea of moving them to their own helper.

Reply:
Ahh, I was unaware of those existing helpers... I'll move them to this file. The build should actually stay green, as we're calling the new file from `base_spec_helper`...